PAC/common/device/device.cpp

```diff
@@ -383,6 +383,11 @@ void dev_stub::on()
     // Ничего не делаем.
     }
 //-----------------------------------------------------------------------------
+bool dev_stub::is_active()
+    {
+    return get_state() == 0 ? false : true;

```

Comment:
Чтобы не было дублирования кода, предлагаю так:
```suggestion
    return i_DI_device::is_active();
```